llama_stack/distribution/server/server.py

```diff
@@ -465,12 +465,19 @@ def main():
         print(f"HTTPS enabled with certificates:\n  Key: {keyfile}\n  Cert: {certfile}")
 
     listen_host = ["::", "0.0.0.0"] if not args.disable_ipv6 else "0.0.0.0"
+    port = return_flag_value(
+        flag="port",
+        env_var="LLAMA_STACK_PORT",
+        args=args,
+        default_args=default_args,
```

Comment:
nit: you could rearrange the order of lines 472-473 to reflect the order of precedence.